docs/guide-zh-CN/README.md

```diff
@@ -1,29 +0,0 @@
-基于Yii 2 的 Twitter Bootstrap 扩展
```

Comment:
You should not remove translations...

Reply:
Sorry, my mistake. I totally agree. Did not know, changes/commits created after creating a pull request are automatically included. Fork is cleaned up now. Pull request should be valid now.